src/Sylius/Bundle/PromotionBundle/Form/Type/Action/PercentFloatType.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Sylius\Bundle\PromotionBundle\Form\Type\Action;
+
+
+use Sylius\Bundle\PromotionBundle\Form\DataTransformer\PercentFloatToLocalizedStringTransformer;
+use Symfony\Component\Form\Extension\Core\Type\PercentType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class PercentFloatType extends PercentType
```

Comment:
I'm wondering, do we need this class? Maybe we can just call `$builder->get('percentage')->addViewTransformer(new PercentFloatToLocalizedStringTransformer());` in `PercentageDiscountConfigurationType`?

Reply:
Yes i removed it and put instead
```php
 $builder->get('percentage')->resetViewTransformers()->addViewTransformer(new PercentFloatToLocalizedStringTransformer());```